cpp/CMakeLists.txt

```diff
@@ -186,17 +186,22 @@ static|shared (default shared)")
   #----------------------------------------------------------------------
   # Thirdparty toolchain options
 
-  # Determine how we will look for dependencies:
+  # Determine how we will look for dependencies
   # * AUTO: Guess which packaging systems we're running in and pull the
   #   dependencies from there. Build any missing ones through the
-  #   ExternalProject setup.
+  #   ExternalProject setup. This is the default unless the CONDA_PREFIX
+  #   environment variable is set, in which case the CONDA method is used
   # * BUNDLED: Build all dependencies through CMake's ExternalProject facility.
   # * SYSTEM: Use CMake's find_package and find_library without any custom
   #   paths. If packages are on non-default locations, let the user indicate it
-  #   from the outside using the *_ROOT variables.
-  #   If your system packages are in general in a non-default location, you can
-  #   set a general default for the *_ROOT variables using ARROW_PACKAGE_PREFIX.
-  # * CONDA: Same as system but set all *_ROOT variables to ENV{CONDA_PREFIX}.
+  #   from the outside using the *_ROOT variables.  If your system packages are
```

Comment:
nit:
```suggestion
  #   using the environment by setting *_ROOT variables.  If your system packages are
```

Reply:
Is this right? What the documentation currently says is to do:

```
cmake .. -DGTest_ROOT=$GTEST_PATH
```

https://github.com/apache/arrow/blob/master/cpp/cmake_modules/ThirdpartyToolchain.cmake#L86

Reply:
Actually both are working in CMake 3.11+, before that only `-DGTest_ROOT=$GTEST_PATH` works.